public/app/plugins/panel/logs/module.tsx

```diff
@@ -33,6 +33,23 @@ export const plugin = new PanelPlugin<Options>(LogsPanel)
           description: '',
           defaultValue: false,
         });
+    } else {
+      builder.addRadio({
```

Comment:
Yeah this works well, but it might be nice to create an issue for core to allow panels to conditionally render options depending on values of other options (I swear I've seen this before but can't seem to find it)